react/components/Table/SimpleTable.js

```diff
@@ -58,6 +61,33 @@ class SimpleTable extends Component {
       : DEFAULT_COLUMN_WIDTH
   }
 
+  addLineActionsToSchema(schema, lineActions) {
+    return (schema.properties = {
+      ...schema.properties,
+      // eslint-disable-next-line camelcase
+      _VTEX_Table_Internal_lineActions: {
+        type: 'any',
+        title: ' ',
+        width: 70,
```

Comment:
⛏this could be a const outside the class like the other static numbers. 
LINE_ACTIONS_COLUMN_WIDTH or something like that.